src/NLog/LayoutRenderers/Wrappers/WrapLineLayoutRendererWrapper.cs

```
@@ -0,0 +1,107 @@
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
+// THE POSSIBILITY OF SUCH DAMAGE.
+// 
+
+namespace NLog.LayoutRenderers.Wrappers
+{
+    using System;
+    using System.ComponentModel;
+    using System.Text;
+
+    using NLog.Config;
+
+    /// <summary>
+    /// Replaces newline characters from the result of another layout renderer with spaces.
+    /// </summary>
+    [LayoutRenderer("wrapline")]
+    [AmbientProperty("WrapLine")]
+    [ThreadAgnostic]
+    public sealed class WrapLineLayoutRendererWrapper : WrapperLayoutRendererBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WrapLineLayoutRendererWrapper" /> class.
+        /// </summary>
+        public WrapLineLayoutRendererWrapper()
+        {
+            WrapLine = 80;
+        }
+
+        /// <summary>
+        /// Gets or sets the line length for wrapping.
+        /// </summary>
+        /// <remarks>
+        /// Only positive values are allowed
+        /// </remarks>
+        /// <docgen category='Transformation Options' order='10' />
+        [DefaultValue(80)]
+        public int WrapLine { get; set; }
+
+        /// <summary>
+        /// Post-processes the rendered message. 
+        /// </summary>
+        /// <param name="text">The text to be post-processed.</param>
+        /// <returns>Post-processed text.</returns>
+        protected override string Transform(string text)
+        {
+            if (WrapLine <= 0)
+            {
+                throw new ArgumentException("WrapLine must be a positive integer");
```

Putting it in the setter will throw an exception :
`NLog.NLogConfigurationException : Error when setting property 'WrapLine' on Layout Renderer: ${wrapline}`
-> will fail faster, on Layout creation, with an exception

Leaving it here will log into InternalLogger. I had a look at other WrappingLayoutRenderers, did not find similar case. 
-> will fail later, more silently, producing an empty string, and logging a warning in internal logger.

An alternative could be something like this:

```
protected override string Transform(string text)
{
    if (WrapLine <= 0)
    {
        InternalLogger.Warn("WrapLine must be a positive integer");
        return text;
    }
```